docker-compose.yml

```diff
@@ -0,0 +1,96 @@
+version: '2'
+
+services:
+  postgres:
+    image: postgres:11.2
+    volumes:
+      - ./data:/docker-entrypoint-initdb.d
+    ports:
+      - "5432:5432"
+  redis:
+    image: redis:latest
+  web:
+    image: notification-api
+    restart: on-failure
+    build:
+      context: ./
+      dockerfile: ci/Dockerfile
+    environment:
+      - REDIS_URL=redis://redis:6379/0
+      - SQLALCHEMY_DATABASE_URI=postgres://postgres:postgres@postgres:5432/postgres
+    command: >
+      bash -c "make generate-version-file && flask run -p 6011 --host=0.0.0.0"
+    volumes:
+      - .:/app
+    ports:
+        - "6011:6011"
+    depends_on:
+      - postgres
+      - redis
+  beat:
+    image: notification-api
+    restart: on-failure
+    build:
+      context: ./
+      dockerfile: ci/Dockerfile
+    environment:
+      - REDIS_URL=redis://redis:6379/0
+      - SQLALCHEMY_DATABASE_URI=postgres://postgres:postgres@postgres:5432/postgres
+    command: >
+      bash -c "sh scripts/run_celery_beat.sh"
+    volumes:
+      - .:/app
+    depends_on:
+      - postgres
+      - redis
+      - web
+  worker:
+    image: notification-api
+    restart: on-failure
+    build:
+      context: ./
+      dockerfile: ci/Dockerfile
+    environment:
+      - REDIS_URL=redis://redis:6379/0
+      - SQLALCHEMY_DATABASE_URI=postgres://postgres:postgres@postgres:5432/postgres
+    command: >
+      bash -c "rm /tmp/celery.pid; sh scripts/run_celery.sh"
```

Comment:
Curious why you need this? Also, don't you need a `rm -f` because this file may not exist?

Reply:
Good question! When testing I found out the pid file was still in there, but it doesn't make sense. The /tmp folder is not a docker volume and should not persist between container up/down. I will do more investigation on this.

Reply:
Alright, I would need more devices to make test, but under linux, when stopping and restarting a container, `/tmp` is not cleared.

As for your other question :

> Also, don't you need a rm -f because this file may not exist?

If I was using `&&` instead of `;`, yes. The `;` operator doesn't care what the output of the previous version is.